src/commcare_cloud/ansible/deploy_commcarehq.yml

```diff
@@ -138,4 +138,4 @@
   hosts: celery
   tasks:
     - include_tasks: roles/commcarehq/tasks/celery_cron.yml
```

Comment:
I tried to run this on india today with

```
cchq india deploy-stack --limit celery --tags cron
```

In order for this task to get picked up I had to change it to `import_tasks`. https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse.html for difference between import_* and include_*.

I'm not sure if that's the right solution, but without it I think the `cron` tags in `celery_cron.yml` might not do anything.

Reply:
@dannyroberts sure, I will make a note of it. thanks

Reply:
I think it is generally preferable to use `import_tasks`

Reply:
@dannyroberts updated files as you suggested. please review and let me know if any changes are necessary.